platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/state/service/ReadableRosterStoreImpl.java

```diff
@@ -95,4 +96,16 @@ public Roster getActiveRoster() {
     public Roster get(@NonNull final Bytes rosterHash) {
         return rosterMap.get(ProtoBytes.newBuilder().value(rosterHash).build());
     }
+
+    /** {@inheritDoc} */
+    @NonNull
+    @Override
+    public List<RoundRosterPair> getRosterHistory() {
```

Comment:
[The roster proposal](https://github.com/hashgraph/hedera-services/blob/16066-tss-roster-update/platform-sdk/docs/proposals/001-tss-rosters-design-proposal/proposal.md#readablerosterstore) has described some additional conditions regarding this i.e.
```
/**
     * Gets the roster history.
     * Returns the active roster history iff:
     *      the roster state singleton is not null
     *      the list of round roster pairs is not empty
     *      the active roster hashes are present in the roster map
     * otherwise returns null.
     * @return the active rosters
     */
```

Reply:
Thanks will update